locale/en/manageiq.po

```diff
@@ -37864,6 +37938,10 @@ msgstr ""
 msgid "IBM Cloud Block-Storage Volume"
 msgstr ""
 
+#: ../app/models/manageiq/providers/ibm_cic/cloud_manager.rb
+msgid "IBM Cloud Infrastructure Center"
```

Comment:
Strange this showed up - Do we normally translate product names?  cc @agrare ?

Reply:
It looks like this is coming from `catalog_types`:
```
  def self.catalog_types
    {"IbmCic" => N_("IBM Cloud Infrastructure Center")}
  end
```

And we do typically mark these for translation e.g.
```
  def self.catalog_types
    {"amazon" => N_("Amazon")}
  end
```

Reply:
Ok, this is fine then